scripts/qa_crowbarsetup.sh

```diff
@@ -2888,6 +2888,11 @@ function deploy_single_proposal()
 {
     local proposal=$1
 
+    # skip a proposal
+    if [ -n "skip_proposal_$proposal" ]; then
```

Comment:
and additionally, you would need to pass the variable from mkcloud into qa_crowbarsetup - maybe similar to want_ vars